fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -229,7 +229,7 @@ fi
 
 DoSliceTimeCorrection=`opts_GetOpt1 "--doslicetime" $@`                  # Whether to do slicetime correction (TRUE), FALSE to omit
 SliceTimerCorrectionParameters=$(opts_GetOpt1 "--slicetimerparams" "$@") # A '@' separated list of FSL slicetimer options. Please see FSL slicetimer documentation for details.
-                                                                         # Verbose (-v) is already turned on. TR (-r) is specified using --tr parameter.
+                                                                         # Verbose (-v) is already turned on. TR is read from the file itself.
```

Comment:
"file" is too generic.  Can we say: "TR is read from 'pixdim4' of the input NIFTI itself".

Reply:
Updated here and in the joint PR.